kotsadm/pkg/handlers/troubleshoot.go

```diff
@@ -605,6 +555,7 @@ func GetSupportBundleRedactions(w http.ResponseWriter, r *http.Request) {
 	JSON(w, 200, getSupportBundleRedactionsResponse)
 }
 
+// NOTE: there is no auth on this route
```

Comment:
here

Reply:
@laverya can you help me with this one? i cant find it called from the ui and it looks like you wrote it

Reply:
This is called by the support bundle command